.github/workflows/matrix-tests.yml

```diff
@@ -21,8 +21,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
+        shell: bash
         run: |
           python -m pip install --upgrade pip
-          pip install -e .[test]
+          SETUPTOOLS_SCM_PRETEND_VERSION=`cat .SETUPTOOLS_SCM_PRETEND_VERSION` pip install -e .[test]
```

Comment:
I was going to use `$(...)`, but I decided backticks were a better choice. Now I don't remember why...
Does `$(...)` work with POSIX Bourne shell? I might have hesitated because of not being sure that would be the case, but maybe that's misguided.

Reply:
OK, no, that was misguided, https://stackoverflow.com/questions/22709371/backticks-vs-braces-in-bash says `$()` is POSIX compliant, and backticks are deprecated! I learned something today.

Reply:
yeah the nice thing about `$()` is that it nests, e.g.
```sh
echo $(echo $(echo foo) $(echo bar))
```